src/pages/workspaces/workspace/Data.js

```diff
@@ -1384,163 +1384,149 @@ const WorkspaceData = _.flow(
             div({ style: styles.tableViewPanel }, [
               _.includes(selectedData?.type, [workspaceDataTypes.entities, workspaceDataTypes.entitiesVersion]) &&
                 h(DataTableFeaturePreviewFeedbackBanner),
-              Utils.cond(
+              Utils.switchCase(
+                selectedData?.type,
                 [
-                  createdBy === getUser()?.email || isGoogleWorkspace,
+                  undefined,
```

Comment:
Why `undefined` here? What would this represent?

Reply:
I think you're seeing an artifact of the whitespace changes - the logic that uses `undefined` here hasn't changed.  Try ignoring whitespace when viewing this PR!